ashley/build.gradle

```diff
@@ -0,0 +1,32 @@
+buildscript {
+  ext {
+    junitPlatformVersion = "1.0.0-M4"
+  }
+  dependencies {
+    classpath "org.junit.platform:junit-platform-gradle-plugin:$junitPlatformVersion"
+  }
+  repositories {
+    mavenCentral()
+  }
+}
+
+dependencies {
+  provided "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
+  provided "com.badlogicgames.ashley:ashley:$ashleyVersion"
+
+  testCompile "org.jetbrains.spek:spek-api:$spekVersion"
+  testCompile 'org.assertj:assertj-core:3.8.0'
+
+  testRuntime "org.junit.platform:junit-platform-launcher:$junitPlatformVersion"
+  testRuntime "org.jetbrains.spek:spek-junit-platform-engine:$spekVersion"
+}
+
+apply plugin: 'org.junit.platform.gradle.plugin'
```

Comment:
We generally keep plugins at the top of `build.gradle` files, you can move it above `dependencies` for consistency.